services/tmc/public/runWorkerTemplate.py

```
@@ -0,0 +1,72 @@
+        await execute()
+        exit()
+    except Exception:
+        t, v, tb = sys.exc_info()
+        frames = traceback.extract_tb(tb)
+        line = frames[-1].lineno if frames else 0
+        tb2 = ["Line " + str(f.lineno) + " in " + f.name + "()" for f in frames[2:]]
+        printError(str(v), type(v).__name__, line, tb2)
+        exit()
+
+
+asyncio.create_task(wrap_execution())
```

Ruff (0.15.2)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)